### PR TITLE
Updated CONTRIBUTING.rst to describe setting up dev environment using…

### PR DESCRIPTION
**Pull request by mfschwartz**
… nox instead of virtualenv

### CODE REVIEW

**Comment by dhermes** [comment]
@jonparrott WDYT?

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -62,8 +62,25 @@ but you can also use a ``virtualenv`` of your own creation.
 
 .. _repo: https://github.com/GoogleCloudPlatform/google-cloud-python
 
-Using a custom ``virtualenv``
-=============================
+Use ``nox``, not ``virtualenv``
+===========================
+
+- To install dependencies into a virtual environment, run the unit test via nox.
```
Comment (tseaver) [inline CONTRIBUTING.rst]: Needs a link to the [nox docs](https://nox.readthedocs.io/en/latest/).

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -72,18 +89,17 @@ Using a custom ``virtualenv``
...
-  ``${HOME}/hack-on-google-cloud-python/${ENV_NAME}`` virtual environment you
-  created above will be referred to as ``${VENV}``. To use the instructions
-  in the steps that follow literally, use::
+- From here on in within these instructions, the virtual environment created
+  above will be referred to as ``${VENV}``. To use the instructions in the steps
+  that follow literally, use::
 
-    $ export VENV=${HOME}/hack-on-google-cloud-python/${ENV_NAME}
+    $ export VENV=${HOME}/hack-on-google-cloud-python/storage/.nox/unit-2-7
+    ${ENV_NAME}
```
Comment (tseaver) [inline CONTRIBUTING.rst]: These two lines should be joined (and the `unit-2-7` bit removed).

**Comment by dhermes** [comment]
@tseaver I'm sure @mfschwartz would be fine if we (the maintainers) just took over this PR and sent the changes directly into the branch rather than dragging him into our more refined preferences.

**Comment by mfschwartz** [comment]
@dhermes - definitely, if you guys want to take over this CL that would be fine. Thanks.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (jonparrott) [reply CONTRIBUTING.rst]: I've often considered making Nox's woodcut a fox and giving it the HTML title attribute of "The Nox Docs Fox".

**Comment by chemelnucfin** [comment]
@mfschwartz Do you mind signing the CLA please?

**Comment by chemelnucfin** [comment]
Closed due to CLA not being signed.  Feel free to sign it and notify us.  Thanks.

**Comment by mfschwartz** [comment]
I somehow missed the request to sign the CLA.
I think there's some confusion here - I'm a Google employee, so there's no need for me to sign a CLA.
Can someone please re-open this pull request and approve it?

**Comment by chemelnucfin** [comment]
Sorry about that!

**Comment by chemelnucfin** [comment]
@mfschwartz I addressed the review comments in #4779, if you are fine with the changes, please consent there and we should be able to merge it.  Thanks.

**Comment by chemelnucfin** [comment]
Thanks @lukesneeringer for verifying the employee status.

@jonparrott @tseaver I'm just going to go ahead and merge this, the review changes are in #4779.  We'll just follow up there.

Thanks @mfschwartz!

**Pull request merged**